packages/web/src/components/collection/desktop/CollectionHeader.tsx

```diff
@@ -137,13 +133,14 @@ export const CollectionHeader = (props: CollectionHeaderProps) => {
   const {
     is_scheduled_release: isScheduledRelease,
     release_date: releaseDate,
-    permalink
+    permalink,
+    is_private: isPrivate
   } = collection ?? {}
   const [artworkLoading, setIsArtworkLoading] = useState(true)
   const [filterText, setFilterText] = useState('')
-  const { spacing } = useTheme()
 
   const hasStreamAccess = access?.stream
+  const shouldShowStats = !isPrivate || isOwner
```

Comment:
why is this condition different from the others?

e.g. `isPublished && variant !== Variant.SMART && (!isPrivate || isOwner)`

Reply:
good call - added `variant`. `isPublished` actually maps back to `!isPrivate`. dumb but not gonna handle that migration now.